### PR TITLE
Transforms: Fix padding bug in cut_into_subsequences

### PR DESCRIPTION
**Pull request by dkrako**
Thanks to the examples in #95 from @theDebbister we could find a bug for the padding in `cut_into_subsequences()`.

Fortunately I never used `keep_padded=True`, but I have added the respective tests to prevent regressions.

### CODE REVIEW

File: src/pymovements/transforms.py [hunk 1]
```diff
@@ -287,30 +285,23 @@ def cut_into_subsequences(
     idx = 0
     for t in range(0, arr.shape[0]):
         for i in range(0, n):
-            # cut out 1000 ms piece of trial t
+            # get piece of respective sequence
```
Comment (SiQube) [inline src/pymovements/transforms.py:288]: if we have a comment here I think this should be more specific. e. g. get piece of sequence with shape (window_size, ...)

File: src/pymovements/transforms.py [hunk 2]
```diff
@@ -287,30 +285,23 @@ def cut_into_subsequences(
...
             arr_tmp = np.expand_dims(arr[t, i * window_size: (i + 1) * window_size, :], axis=0)
 
-            # concatenate pieces
+            # fill respective instance with piece
```
Comment (SiQube) [inline src/pymovements/transforms.py:291]: maybe part of the sequence?

File: src/pymovements/transforms.py (hunk 1 above)
Comment (SiQube) [reply src/pymovements/transforms.py:288]: or part of the sequence / subsequence

**Comment by dkrako** [comment]
The function itself is implemented very ineffienctly, so I have created an issue for this #144

**Review by SiQube — APPROVED** [review]

**Pull request merged**